src/components/schedule-visualizer/lift/lift.tsx

```diff
@@ -0,0 +1,114 @@
+import { makeStyles } from '@material-ui/core';
```

Comment:
There is another `lift.tsx` at `src/components/schedule-visualizer/lift.tsx`, maybe something went wrong during the merge.

Reply:
Yeah, it looks like it. I guess I'll stick to rebase and cherry pick. =/